python/orca/src/bigdl/orca/learn/pytorch/pytorch_pyspark_estimator.py

```diff
@@ -154,7 +155,8 @@ def __init__(
             model_dir=self.model_dir,
             log_to_driver=self.need_to_log_to_driver,
             driver_ip=self.ip,
-            driver_port=self.port)
+            driver_port=self.port,
```

Comment:
Shall we rename `driver_port` to `driver_log_port`, `tcp_port` to `driver_tcp_store_port`, or other names you prefer?